jekyll/_ccie/overview.md

```diff
@@ -37,10 +56,10 @@ The first is a services box which contains the CircleCI frontend and all interna
 | Builder Boxes               | all traffic / all ports | Internal Communication |
 | GitHub (Enterprise or .com) | 80, 443                 | Incoming Webhooks      |
 
-#### The Builder Boxes
----
 
-Our Builder Boxes handle running your builds, and store no state themselves. Each builder machine reserves 2CPU/4G for coordinating builds, and then uses the remaining space to create build containers. The larger machine, the more containers it will run. See our [configuration doc]({{site.baseurl}}/enterprise/config/) for more information about how many containers a particular machine can run. Since they store no state, they can be scaled up or down at will. When shutting machines down, be sure to use the `circle-shutdown` command to a gracefully shut down the machine.
+The machines on which Builder instances run may be restarted by using the `circle-shutdown` command to gracefully shutdown. The Builder instances run without storing state, enabling you to increase or decrease containers as needed. To ensure that there are enough builder machines running to handle all of the builds, track the queued builds and increase the Builder instance machines as needed to balance the load.
```

Comment:
Also, if this is intended as an overview document the detail about `circle-shutdown` feels a bit too granular, but perhaps more importantly - it lacks context to know where that command lives and how you execute it.